recipes/fortran-binary/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "fortran-binary" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: be613276fb3141b81933c508990b8fa5e71f1df04a7423567d2aa78a16267241
+
+build:
+  number: 1
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - numpy
+    - python >=3
+
+test:
+  imports:
+    - fortran_binary
```

Comment:
Should the `forbin` command be tested as well (at least a --help or --version). Also I'd recommend changing upstream to use a console_scripts entrypoint instead of scripts in the setuptools metadata.